deploy/template.yaml

```diff
@@ -176,6 +176,10 @@ Resources:
       Environment:
         Variables:
           ENVIRONMENT: !Sub "${Environment}"
+          CLIENT_1_SIGNING_CERT_PARAM: !Sub "/${Environment}/cri/passport/config/client_1/signing_cert"
```

Comment:
I'd remove the client_1, client_2, and client_3 ones. We don't need them right now so probably best to just stick with the test one for now

Reply:
just kept "test" client